.github/workflows/test-upgrade-legacy.yaml

```diff
@@ -0,0 +1,160 @@
+---
+name: "Upgrade PE with one legacy compiler"
```

Comment:
as a followup to https://github.com/puppetlabs/puppetlabs-peadm/pull/339: I think we should stick with single quotes. If someone demands double quotes I think the other files should be adjusted as well.

Reply:
Merging into legacy compiler feature branch. This is logged and will to be addressed at later point.